agent_build_refactored/managed_packages/steps/download_build_dependencies/download_build_dependencies.sh

```diff
@@ -77,27 +123,28 @@ eUy2pBqS1302sBB+nfrGFTFmJC4ypzY8uQFG98dxBozqME836KrMhiBbeLIghXgs
 OQHHt+gXyQVzvzskedUxZ4mxzZ1D/+JfJhwxGvxZFn5sAGLbAmRbXoIv0duHk/JW
 2F1vt/S03rzec8y37B3LLWnIO9LK
 =V4Re
------END PGP PUBLIC KEY BLOCK-----"
+-----END PGP PUBLIC KEY BLOCK-----" | gpg --import
 
-gpg --import <<< ${XZ_PUBLIC_KEY}
```

Comment:
Just thinking out loud - do we need / want to clear gpg data directory before validating each artifact? So we ensure that only key for a specific package is in the gpg key database once verifying that artifact.